syntax = "proto3";




service MLService {
    rpc ProcessPhoto (PhotoRequest) returns (PhotoResponse);
}

message Tag {
    string name = 1;
}

message PhotoRequest {
    int64 id = 1;
    string promt = 2;
    int64 height = 3;
    int64 widht = 4;
    string goal = 5;
    repeated Tag tags = 6;
}


message PhotoResponse {
    int64 id = 1;
    string s3_url = 2;
    string status = 3;
}
